[Bug]  V1.81.0 Codegen Error

### Describe the bug

Hi  :) well I just wanted to let you know that for some reason the latest codegen fails but using either 1.79.0 or 1.80.1 seems to work as expected. I have not yet had time to look into it myself. But I will report back if I at some  point know  why :) 

### Codegen logs with `RUST_LOG=debug` environment variable

```shell
2023/09/10 11:46:07 [ERROR] panicked at 'called `Result::unwrap()` on an `Err` value: Error("lex error")', .cargo\registry\src\index.crates.io-6f17d22bba15001f\flutter_rust_bridge_codegen-1.81.0\src\parser\source_graph.rs:359:83
```


### To Reproduce

_No response_

### Expected behavior

_No response_

### Generated binding code

_No response_

### OS

_No response_

### Version of `flutter_rust_bridge_codegen`

_No response_

### Flutter info

_No response_

### Version of `clang++`

_No response_

### Version of `ffigen`

_No response_

### Additional context

_No response_

--- a/frb_codegen/src/commands.rs
+++ b/frb_codegen/src/commands.rs
@@ -318,7 +318,6 @@ fn run_cargo_expand(dir: &str) -> String {
             // remove first and last line to get rid of wrapping module
             let mut output = stdout.lines();
             output.next();
-            output.next_back();
             output
                 .collect::<Vec<_>>()
                 .join("\n")

